EssaGUI/gui/Text.hpp

```diff
@@ -1,16 +1,14 @@
 #pragma once
 
-#include "LLGL/Resources/TTFFont.hpp"
-namespace GUI{
-
-class Text{
-public:
+#include <LLGL/Resources/TTFFont.hpp>
 
+namespace GUI {
 
+class Text {
+public:
 private:
     llgl::TTFFont m_font;
     int m_font_size;
 };
 
 }
```

Comment:
kind of dead code waiting for its moment

Reply:
removed after almost 3 months